Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/Scripts/userSetup.py

```diff
@@ -245,20 +301,23 @@ def post_startup():
     # Lumberyard DCCsi environment ready or error out.
     try:
         import azpy.dcc.maya
-        _LOGGER.info('Python module imported: azpy.dcc.maya')
+        from azpy.dcc.maya import _PACKAGENAME
+        _LOGGER.info('Python module imported: {}'.format(_PACKAGENAME))
     except Exception as e:
         _LOGGER.error(e)
         _LOGGER.error(traceback.print_exc())
         return 1
 
-    # Dccsi azpy maya ready or error out.
-    try:
-        azpy.dcc.maya.init()
-        _LOGGER.info('SUCCESS, azpy.dcc.maya.init(), code accessible.')
-    except Exception as e:
-        _LOGGER.error(e)
-        _LOGGER.error(traceback.print_exc())
-        return 1
+    # DEPRECATE: don't nee to .init() anymore, auto-init (experimental)
```

Comment:
Remove

Reply:
I'll remove in next PR, this is here to remind me of this deprecation pass as this change was made within this PR